central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl.go

```diff
@@ -58,6 +59,7 @@ type managerImpl struct {
 	deployments     deploymentDataStore.DataStore
 	images          imgDataStore.DataStore
 	imageCVEs       imgCVEDataStore.DataStore
+	imageCVE2       imageCVE2DataStore.DataStore
```

Comment:
nit:  imageCVEsV2

I'm going to leaving these.  Just basically same comment throughout the PR.